README.md

```diff
@@ -59,3 +59,17 @@ Once you have downloaded the source files then the package can be installed by r
     flit install
 
 (To learn the basics about setting up Git see [this link](https://help.github.com/articles/set-up-git/).)
+
+## Citation
+
+QuantEcon.py is MIT licensed, so you are free to use it without any charge and restriction. If it is convenient for you, please cite QuantEcon.py when using it in your work and also consider contributing all your changes back, so that we can incorporate it.
+
+A BibTeX entry for LaTeX users is
+
+```bibtex
+@article{10.21105/joss.05585,
+author = {Batista, Quentin and Coleman, Chase and Furusawa, Yuya and Hu, Shu and Lunagariya, Smit and Lyon, Spencer and McKay, Matthew and Oyama, Daisuke and Sargent, Thomas J. and Shi, Zejin and Stachurski, John and Winant, Pablo and Watkins, Natasha and Yang, Ziyue and Zhang, Hengcheng},
+doi = {10.5281/zenodo.10345102},
+title = {QuantEcon.py: A community based Python library for quantitative economics}
```

Comment:
`year` and `journal` fields are required?

```suggestion
title = {QuantEcon.py: A community based Python library for quantitative economics},
year = {2024},
journal = {Journal of Open Source Software},
volume = {9},
number = {93},
pages = {5585}
```

Reply:
Thanks! added them. I used `cffconvert` to convert cff to bibtex.